### PR TITLE
Wip/query map solr

### PR DESCRIPTION
**Pull request by arnabGudu**

### CODE REVIEW

File: api/service.py [hunk 1]
```diff
@@ -2,9 +2,11 @@
 import pysolr
 from kafka import KafkaConsumer
 import threading
+import argparse
 
 solr_url = 'http://localhost:8983/solr/ondc'
 solr = pysolr.Solr(solr_url, always_commit=True)
+solr_qm = pysolr.Solr('http://localhost:8984/solr/query_map', always_commit=True)
```
Comment (kLamda) [inline api/service.py:9]: Add a configuration file having the URL.

File: api/app.py [hunk 2]
```diff
@@ -121,6 +80,149 @@ def search_handler(query):
...
+
+    results = solr.search("*:*", fq=solr_query, rows=100)
+    return results, solr_query, mod_query
+
+
+@app.get("/search/{rawQuery}", status_code=status.HTTP_200_OK)
+async def searchProduct(rawQuery: str, f: str = None):
+    try:
+        print('rawQuery', rawQuery)
+        results, solr_query, mod_query = search_handler(rawQuery, f)
```
Comment (kLamda) [inline api/app.py:204]: A provision to pull "All" products from DB and send them to Frontend.

**Review by kLamda — CHANGES_REQUESTED** [review]

File: api/service.py (hunk 1 above)
Comment (arnabGudu) [reply api/service.py]: Done

File: api/app.py (hunk 2 above)
Comment (arnabGudu) [reply api/app.py]: Done

**Review by kLamda — APPROVED** [review]
Review done

**Pull request merged**